scania-digital-design-system/sdds
# fix(dropdown): refactored remove/adding to array and removed sort function.

**Describe pull-request**  
Removed the push/splice methods and moved over to functional programming, map/filter and creating new arrays. Also removed the sort function since it did nothing and emptied the array.

**Solving issue**  
Fixes: -

**How to test**  
1. Go to https://deploy-preview-620--tegel-storybook.netlify.app/ (the latest working deploy)
2. Check in Components -> Dropdown
3. Check that the multiselect and default options all works as intended. 

**Screenshots**  

**Additional context**
